opentok/opentok.py

```diff
@@ -359,7 +364,7 @@ def start_archive(self, session_id, has_audio=True, has_video=True, name=None, o
         elif response.status_code == 403:
             raise AuthError()
         elif response.status_code == 400:
-            raise RequestError("Session ID is invalid")
+            raise RequestError("Session ID is invalid or resolution error")
```

Comment:
I think if possible we need to determine which factor caused this error and raise a specific error.

Reply:
Yeah. Cannot agree more. However, I have triggered resolution error (set resolution different string or other type of value), the status_code of response is 400 as well. In addition, the document of OpenAPI also describe the status_code will be 400. I think the python SDK only can determine the error after the OpenAPI respond a different status_code.